configure.ac

```diff
@@ -548,6 +548,7 @@ AH_VERBATIM([__STDC_FORMAT_MACROS],
 PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.28], [], [AC_ERROR([glib-2.0 (version 2.28+) has to be installed!])])
 PKG_CHECK_MODULES([VMI], [libvmi >= 0.11], [], [AC_ERROR([LibVMI not found])])
 PKG_CHECK_MODULES([JSONC], [json-c], [], [AC_ERROR([libjson-c not found])])
+PKG_CHECK_MODULES([ZLIB],[zlib], [], [AC_ERROR([zlib not found])])
```

Comment:
So since this is an optional package thats only used by one plugin, I don't think we should AC_ERROR when its not found. Should only check for it when the plugin that uses it is enabled.

Reply:
Done.